packages/cli/src/commands/sync.ts

```diff
@@ -36,7 +36,16 @@ export const sync = async () => {
     return;
   }
 
-  const { host, token } = globalConfig[localConfig.projectId];
+  const projectConfig = globalConfig[localConfig.projectId];
+
+  if (projectConfig === undefined) {
+    spinner.fail(
+      `Project config is not found. Please make sure the project is linked. \n Use webstudio link command to link your project`
```

Comment:
`Project config is not found, please run "webstudio-cli link".

Also would be good to consistently use color for commands, example from init: `      `Run ${pc.dim("npx vercel")} to publish on Vercel.`,`